tests/phpunit/tests/term/meta.php

```diff
@@ -199,7 +197,7 @@ public function test_lazy_load_term_meta_false() {
 		foreach ( $terms as $t ) {
 			add_term_meta( $t, 'foo', 'bar' );
 		}
-
+		$this->reset_lazyload_queue();
```

Comment:
This call also reset comment queue `$lazyloader->reset_queue( 'comment' );` Don't think it will impact anywhere.

Reply:
I don't think this is a problem here.